### PR TITLE
 Add post about NewRelic and Bref

### PR DESCRIPTION
**Pull request by Nyholm**
@aradoje could you review please?

### CODE REVIEW

File: _posts/2019-06-21-newrelic-on-bref-aws-lambda.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+#!/bin/sh
+
+# Start NewRelic daemon
+/opt/bref/sbin/newrelic-daemon -c /opt/bref/etc/newrelic/newrelic.cfg
+
+# Start PHP-FPM
+/opt/bootstrap-php
+{% endhighlight %}
+
+Now I add a line in `bootstrap-php` to make sure NewRelic is not consider my requests as background jobs:
```
Comment (aradoje) [inline _posts/2019-06-21-newrelic-on-bref-aws-lambda.md]: NewRelic is not consider*ing* my requests?

**Comment by aradoje** [comment]
Maybe you could add also the part about publishing the layer and changing the runtime in Lambda to use the new layer.

File: _posts/2019-06-21-newrelic-on-bref-aws-lambda.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  echo "loglevel=error" > ${INSTALL_DIR}/etc/newrelic/newrelic.cfg && \
+  echo "logfile=/dev/null" >> ${INSTALL_DIR}/etc/newrelic/newrelic.cfg
+
+{% endhighlight %}
+
+I disable logging since I run this on a read-only environment.  
+
+This is great and it should have been working on a normal server. However, NewRelic is configured to automatically start
+with the PHP-FPM server. Since Bref has it custom implementation of PHP-FPM that will not work. We need to start the the
+newrelic-daemon manually every time we load or Lambda function.
```
Comment (carlos-ea) [inline _posts/2019-06-21-newrelic-on-bref-aws-lambda.md]:
```suggestion
newrelic-daemon manually every time we load our Lambda function. 
```

File: _posts/2019-06-21-newrelic-on-bref-aws-lambda.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                - 'arn:aws:lambda:eu-central-1:123456789:layer:bref-php-73-fpm:1'
+{% endhighlight %}
+
+When everything is deployed, you need to make sure that your lambda function can reach internet. The NewRelic 
+daemon must be able to send data to NewRelic's servers. You should also add `newrelic_start_transaction` and 
+`newrelic_end_of_transaction` on appropriate places in your application. Or else NewRelic will log really 
+long response times. If you are using Symfony and EkinoNewRelic bundle, you should configure it as you were
+using Symfony HTTP Cache. 
+
+To accomplish this setup took me way more hours then I want to admit. So I hope this small post could help
```
Comment (carlos-ea) [inline _posts/2019-06-21-newrelic-on-bref-aws-lambda.md]:
```suggestion
To accomplish this setup took me way more hours than I want to admit. So I hope this small post could help 
```

**Pull request merged**